src/contracts/core/DelegationManager.sol

```diff
@@ -797,13 +807,36 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         }
     }
 
-    function _initializeWithdrawalDelayBlocks(uint256 _withdrawalDelayBlocks) internal {
+    /**
+     * @notice Sets the withdrawal delay blocks for each strategy in `_strategiesToSetDelayBlocks` to `_withdrawalDelayBlocks`.
+     * only gets called once when initializing the contract.
+     */
+    function _initializeStrategyWithdrawalDelayBlocks(
+        IStrategy[] calldata _strategiesToSetDelayBlocks,
+        uint256[] calldata _withdrawalDelayBlocks
+    ) internal {
```

Comment:
I wonder if it's worth it to add a function that allows us to update withdrawal delay blocks per strategy?

I think now that we're not doing a global delay, this might be a good idea, in case we misconfigure a strategy?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/392/commits/323fa4e387941179468d541a4c9d9470c2382bd1#diff-d323417ded59a593f4ea410225a9167b24198c4d2c05cbc0fe9af47d34b88810R511
Added setter here